src/main/java/io/github/cottonmc/energy/api/ElectricalEnergyType.java

```diff
@@ -1,20 +1,35 @@
 package io.github.cottonmc.energy.api;
 
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
+
 public class ElectricalEnergyType implements EnergyType {
 
-    private final int size;
+    private final int lvPerPacket;
 
-    public ElectricalEnergyType(int maxSize) {
-        this.size = maxSize;
+    public ElectricalEnergyType(int lvPerPacket) {
+        this.lvPerPacket = lvPerPacket;
```

Comment:
Why are you setting lvPerPacket to maxSize? Since you will handle different energy levels below by getDisplayAmount() you might as well set the max size to Integer.MAX_INT (or whatever it is called in java)

Reply:
I'm not setting lvPerPacket to maxSize. The semantics of the constructor argument have changed.